docs/eventing/getting-started.md

```diff
@@ -0,0 +1,441 @@
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative Eventing installed](./install/index.html). 
+
+## Setting up Eventing subcomponents
+
+Before you start to send events, you need to create the components needed to move the events. This guide walks you through creating each Eventing subcomponent.
+
+### Configuring namespace for eventing
+
+In most Knative tutorials, you will operate inside of a namespace. A namespace groups together subcomponents so that you can easily organize them.
+
+1. Create the namespace. This command creates a namespace called **event-example**:
+
+```sh
+kubectl create namespace event-example
+```
+
+2. Set up the namespace for Knative Eventing. To set up the namespace, add a label to your namespace with this command:
+
+```sh
+kubectl label namespace event-example knative-eventing-injection=enabled
+```
+
+The `knative-eventing-injection` label triggers Knative to add Knative Eventing subcomponents such as the `Broker` in your namespace.
+
+In the next section, you'll learn more about the `Broker`.
+
+### Validating that the `Broker` is running
+
+The `Broker` ensures that every event sent by event producers is sent to the correct event consumers. While you created the `Broker` when you labeled your namespace as ready for eventing, it is important to verify that your `Broker` is working correctly. In this guide, you will use the default broker.
+
+1. Use the following command to verify that the `Broker` is in a healthy state:
+
+```sh
+kubectl -n event-example get Broker default
+```
+
+This should return the following:
+
+```sh
+NAME      READY   REASON   HOSTNAME                                                           AGE
+default   True             default-Broker.event-example.svc.cluster.local   1m
+```
+
+When the `Broker` is **READY**, it can begin to manage the events it receives.
+
+2. If the **READY** column reads **False**, wait 2 minutes and repeat Step 1. If the **READY** column still reads **False**, see the [Debugging Guide](./debugging/README.md) to troubleshoot the issue.
+
+Now your `Broker` is ready to manage your events.
+
+### Creating event consumers
+
+These subcomponents receive the events sent by event producers (you'll create those a little later). You'll create two event consumers, `hello-display` and `goodbye-display`, so that you can see how to selectively send events to different consumers later.
+
+
+1. To create the `hello-display` subcomponent, enter the following command:
+
+```sh
+kubectl -n event-example apply -f - << END
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello-display
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: hello-display
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      containers:
+        - name: event-display
+          # Source code: https://github.com/knative/eventing-contrib/blob/release-0.6/cmd/event_display/main.go
```

Comment:
have these images been updated for 0.7 release?

Reply:
When I spoke with @akashrv, he believed that was the current image. I'll go ahead and confirm if this is still the case.

Reply:
It is an image of event-display container. It shouldn't be affected by Knative eventing 0.7 release and should work across releases.